android/app/src/main/java/com/rohtvapp/ROHBitMovinPlayerPackage/ROHBitMovinPlayerManager.java

```diff
@@ -224,7 +236,7 @@ public void setConfiguration(PlayerView view, ReadableMap config) {
         poster = configuration.getString("poster");
       }
 
-      List subtitleTracks = Arrays.<SubtitleTrack>asList(subtitleTrack);
+      //List subtitleTracks = Arrays.<SubtitleTrack>asList(subtitleTrack);
```

Comment:
As above

Reply:
it is a WIP, about subtitles. I will remove it later.